src/components/SelectedProjects/EditSelectedProject.jsx

```diff
@@ -95,6 +96,11 @@ export default function EditSelectedProject({ pageInfo }) {
         }, body, id)
     }
 
+    async function deletePhoto(photoId) {
+        await deleteProjectPhotos(photoId);
+        history.go(0);
```

Comment:
Don't need to reload page. You must refetch updated data or you can remove it local from state.